catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/model/Search.java

```diff
@@ -361,6 +363,15 @@ private void addObject(Map<String, Object> obj, String name, Object value) {
             metacard.put(CACHED, ISO_8601_DATE_FORMAT.print(new DateTime()));
         }
 
+        Attribute internalLocalResource = result.getMetacard()
```

Comment:
While not radically better, this might be a bit clearer:
```java
Optional<Serializable> internalLocalResource = Optional.ofNullable(result.getMetacard()
        .getAttribute(INTERNAL_LOCAL_RESOURCE))
        .map(Attribute::getValue);
if (internalLocalResource.isPresent()) {
    metacard.put(INTERNAL_LOCAL_RESOURCE, internalLocalResource.get());
} else {
    metacard.put(INTERNAL_LOCAL_RESOURCE, Boolean.FALSE);
}
```